coriolis/tests/test_cache.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+import unittest
+from unittest import mock
+from coriolis import exception
+from coriolis import cache
+
+
+class TestGetCacheDecorator(unittest.TestCase):
+
+    @mock.patch.object(cache.cache, 'get_memoization_decorator')
+    def test_get_cache_decorator(self, mock_get_memoization_decorator):
+        mock_get_memoization_decorator.return_value = 'ExpectedValue'
```

Comment:
Just for future reference: you don't have to necessarily assign a value to `return_value` in this case. 
You can simply:
```python
self.assertEqual(result, mock_get_memoization_decorator.return_value)
```

Reply:
Perfect! Good to know for the future! Thanks!